superset/charts/schemas.py

```diff
@@ -725,20 +727,20 @@ class ChartDataQueryContextSchema(Schema):
     )
     result_type = fields.String(
         description="Type of results to return",
-        validate=validate.OneOf(choices=("full", "query", "results", "samples")),
+        validate=validate.OneOf(choices=("query", "results", "samples")),
```

Comment:
`full` should be a valid choice here, see for instance https://github.com/apache/incubator-superset/blob/master/superset-frontend/src/chart/chartAction.js#L343-L350